### ISSUE TITLE
Compiling issue

### ISSUE THREAD

**Issue by pierrotm777** [panStamp/swap#2]
After an update your swap library return two errors:
# include "storage.h" into register.h (storage.h not found)
# include "ccpacket.h" into swpacket.h (ccpacket.h not found)

**Comment by dberenguer** [comment]
Did you install the panStamp cores from Tools->Boards?

**Comment by pierrotm777** [comment]
By the Board manager !

**Comment by dberenguer** [comment]
Yes, from the Boards manager. Your errors seem to be reporting that some core files are not installed. Otherwise, try to do a fresh install.
